arithmetization/src/test/java/net/consensys/linea/zktracer/module/mmu/RevertingLogsTests.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer.module.mmu;
+
+import static net.consensys.linea.testing.BytecodeCompiler.newProgram;
+
+import java.util.List;
+
+import net.consensys.linea.testing.ToyAccount;
+import net.consensys.linea.testing.ToyExecutionEnvironmentV2;
+import net.consensys.linea.testing.ToyTransaction;
+import net.consensys.linea.zktracer.opcode.OpCode;
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.crypto.KeyPair;
+import org.hyperledger.besu.crypto.SECP256K1;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.datatypes.Hash;
+import org.hyperledger.besu.datatypes.TransactionType;
+import org.hyperledger.besu.datatypes.Wei;
+import org.hyperledger.besu.ethereum.core.Transaction;
+import org.junit.jupiter.api.Test;
+
+public class RevertingLogsTests {
+
+  private static final Bytes CALLDATACOPY =
+      newProgram()
+          .op(OpCode.CALLDATASIZE) // size
+          .push(0) // offset
+          .push(0) // dest offset
+          .op(OpCode.CALLDATACOPY)
+          .compile();
```

Comment:
The purpose of this byte code chunk is to populate memory. Maybe let the name reflect that, e.g. calling it
```rust
POPULATE_MEMORY
```
Or so. Also you could achieve the same goal more reliably with a `PUSH32` / `MSTORE` combo (reliable in the sense that you don't need to remember to provide the transaction with a payload.) We do BTW have `populateMemory` methods at various places.